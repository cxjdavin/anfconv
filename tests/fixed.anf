c RUN: %solver --anfread %s --nodefault --printfinal | %check %s.out
x1
