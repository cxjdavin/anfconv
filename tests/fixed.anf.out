c CHECK-L: Fixed values
c CHECK-NOT-L: (skip stuff)
c CHECK-L: x(1)
